### PR TITLE
Score specific values for line spacing parameters

### PR DESCRIPTION
**Pull request by rpspringuel**
Addresses #1038.

Default values have been chosen based on a 12pt article.  Let me know if you want these tweaked.

### CODE REVIEW

**Comment by henryso** [comment]
I'm assuming no tests changed?

File: tex/gregoriotex-spaces.tex [hunk 1]
```diff
@@ -1457,6 +1473,9 @@
     {spaceporrectuspunctummorashift}{\gre@rubberfalse}%
     {raresignshift}{\gre@rubberfalse}%
     {divisiofinalissep}{\gre@rubberfalse}%
+    {lineskip}{\gre@rubberfalse}%
```
Comment (eroux) [inline tex/gregoriotex-spaces.tex]: Why using non-stretchable dimen instead of skips?

Comment (rpspringuel) [inline tex/gregoriotex-spaces.tex]: The default values for those three led me to believe they were dimensions, but after a bit more research I realize they are skips whose stretch and shrink are 0 by default.  I'll fix that.

**Comment by rpspringuel** [comment]
@henryso no tests change.

**Comment by rpspringuel** [comment]
So I've modified a test to show what increasing the parameters does, though I make no claim that said modification attempts to be systematic about it.

**Pull request merged**